emperor/support_files/js/color-view-controller.js

```diff
@@ -182,6 +186,16 @@ ColorViewController.getDiscreteColor = function(index, map){
   return ColorViewController._discreteColormaps[map][index];
 };
 
+/* Change the color for a set of plottables - group: list of plottables */
```

Comment:
Can you document the arguments to this function and what the function does, also it would be good to change `scope` to something like `dv` (for `DecompositionView`) or something like that.